[2.9 ACRA] Resource Formatter blocks apkg import in Hungarian?

###### Reproduction Steps

1.  Have locale set to 'hu_HU'
2.  Import .apkg file


###### Expected Result

Successful import


###### Actual Result

```
java.lang.RuntimeException: An error occurred while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:365)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
at java.util.concurrent.FutureTask.run(FutureTask.java:271)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:257)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
at java.lang.Thread.run(Thread.java:784)
Caused by: java.util.IllegalFormatFlagsException: Flags = ' '
at java.util.Formatter$FormatSpecifier.checkText(Formatter.java:3131)
at java.util.Formatter$FormatSpecifier.<init>(Formatter.java:2824)
at java.util.Formatter$FormatSpecifierParser.<init>(Formatter.java:2624)
at java.util.Formatter.parse(Formatter.java:2557)
at java.util.Formatter.format(Formatter.java:2504)
at java.util.Formatter.format(Formatter.java:2458)
at java.lang.String.format(String.java:2814)
at android.content.res.Resources.getString(Resources.java:529)
at com.ichi2.libanki.importer.Anki2Importer.publishProgress(Anki2Importer.java:4)
at com.ichi2.libanki.importer.AnkiPackageImporter.run(AnkiPackageImporter.java:1)
at com.ichi2.async.DeckTask.doInBackgroundImportAdd(DeckTask.java:7)
at com.ichi2.async.DeckTask.doInBackground(DeckTask.java:38)
at com.ichi2.async.DeckTask.doInBackground(DeckTask.java:1)
at android.os.AsyncTask$2.call(AsyncTask.java:345)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
```


https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/report-details/4707c4a6-93d4-4cf4-a35d-2a1d1107ffe0

Looks like there is an illegal string in the hu_HU translations that are used in Anki2Importer.publishProgress?

General question: should we do better about eating these types of exceptions? Or shouldn't there be some sort of translation-string verification tool so that this is impossible? Seems like it should be impossible

--- a/AnkiDroid/src/main/res/values-hu/03-dialogs.xml
+++ b/AnkiDroid/src/main/res/values-hu/03-dialogs.xml
@@ -100,7 +100,7 @@
     <string name="rebuild_custom_study_deck">Egyéni gyakorlópakli újraépítése.\nKérlek, várj...</string>
     <string name="importing_collection">Gyűjtemény importálása.\nKérlek, várj&#8230;</string>
     <string name="import_media_count">Média fájlok importálása %1$d%%\nKérlek, várj&#8230;</string>
-    <string name="import_progress">• Importált megjegyzések: %1$d%%\n• importált kártyák: %2$d%%\n• utómunka: %3$d% %</string>
+    <string name="import_progress">• Importált megjegyzések: %1$d%%\n• importált kártyák: %2$d%%\n• utómunka: %3$d%%</string>
     <string name="import_update_details">Frissített %1$d %2$d meglévő jegyzetek.</string>
     <string name="import_update_ignored">Néhány frissítést figyelmen kívül hagytak, mert a jegyzet típusa megváltozott</string>
     <string name="import_complete_count">Importált kártyák: %d</string>

